lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -66,11 +67,15 @@ class CheckExistingIdentityHandlerTest {
 
     private static final String TEST_FEATURE_SET = "test-feature-set";
     private static final String TEST_CLIENT_OAUTH_SESSION_ID = SecureTokenHelper.generate();
+
+    private static final String TEST_JOURNEY = DCMAW_CRI;
```

Comment:
As it's a requirement that the Step Function passes the full URL e.g. `/journey/cri/build-oauth-request/dcmaw` in the `journey` parameter on the Lambda when running in either API Gateway or Step Function the TEST_JOURNEY should be changed to `/journey/cri/build-oauth-request/dcmaw` in order to reflect that and the code should be able to handle this.

Reply:
We are obtaining the 'criId' from path variable  /journey/cri/build-oauth-request/{criId}, within the API Gateway and setting it in the journey field. We retrieve the configurations from the system manager based on thi criId. I'am not sure how to obtain the full path. If we do get the full path, how do we extract criId from it? I truly don't understand this detail in the task description and task requirements. Please can discuss about that.